src/components/sequencing/form/SelectedCommand.svelte

```diff
@@ -181,6 +184,21 @@
     return hasAncestorWithId(element.parentElement, id);
   }
 
+  function formatTypeName(s: string) {
+    // add spaces to CamelCase names, 'GroundEvent' -> 'Ground Event'
+    return s.split(/(?=[A-Z])/).join(' ');
```

Comment:
Should this be `s.replace(/([^A-Z])(?=[A-Z])/g, '$1 ')` instead? This would handle the case for if the types have consecutive uppercase letters next to each other. Not sure if that's a possibility but this should handle that case.

Reply:
Addressed in [be5f37e](https://github.com/NASA-AMMOS/aerie-ui/pull/1362/commits/be5f37ee41a143a7976f9fec4182680e1c2d4594)